doc/testing.md

```diff
@@ -1,5 +1,12 @@
 # Testing
 
+## Notification calls
```

Comment:
This is not a testing specification. The test spec shall be accurate so the result can be reproduced.
Please add the exact steps to make it work. (Pausing/unpausing car).